packages/std-client/src/setup/createSystemCallStreams.ts

```diff
@@ -0,0 +1,52 @@
+import { SystemCall } from "@latticexyz/network";
+import { Components, World, Type, EntityID, getComponentValue, Component } from "@latticexyz/recs";
+import { Contract, BigNumber } from "ethers";
+import toLower from "lodash/toLower";
+import compact from "lodash/compact";
+import { Subject } from "rxjs";
+import { DecodedSystemCall } from "./types";
+import { createDecodeNetworkComponentUpdate } from "./utils";
+
+export function createSystemCallStreams<C extends Components, SystemTypes extends { [key: string]: Contract }>(
```

Comment:
this is the same logic as before, just pulled out into its own file so I could see it better